.gitignore

```diff
@@ -0,0 +1,3 @@
+#node_modules
+.github/workflow/linters.yml
+test.md
```

Comment:
- [ ] Kindly correct the path to the `test.md` file, we could use as follow 👉🏽 `./test.md` (**with a dot** at the beginning)